README.md

```diff
@@ -1,57 +1,72 @@
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-# d.inspectEHR
+# ccaa.inspectEHR
 
 <!-- badges: start -->
 
 [![Lifecycle:
 experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://www.tidyverse.org/lifecycle/#experimental)
 <!-- badges: end -->
 
-The goal of d.inspectEHR is to provide an early phase light touch
-quality evaluation for the DECOVID project. Prior to running these
-reports, sites *must* apply the DECOVID ddls and *should* try to run
-ACHILLES against their OMOP database.
+The aim of ccaa.inspectEHR is to provide a high-level profile, and data quality evaluation of a critical care database in OMOP format. The package produces 2 html reports, one describing the data available, and one applying a set of data quality checks based on the [Khan framework](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5051581/).
 
-## Installation
+The package is based on a [simillar quality evaluation](https://github.com/DocEd/d.inspectEHR) for the DeCovid project. 
 
-You can install the development version of d.inspectEHR from
-[GitHub](https://www.github.com/DocEd/dinspectEHR) with:
+Users must have an SQL database conforming to the OMOP v5.4 specifications. 
 
-``` r
-remotes::install_github("DocEd/d.inspectEHR")
-```
-
-## Running the Report
+### Installation
 
-The report can be run by loading the package and running the following
-code.
+You can install the development version of ccaa.inspectEHR from
+[GitHub](https://github.com/NICST-PROTECT/ccaa.inspectEHR) with:
 
 ``` r
-library(d.inspectEHR)
-render_report(output = "DECOVID_report.html")
+remotes::install_github("NICST-PROTECT/ccaa.inspectEHR")
 ```
 
-This will prompt a shiny interface to launch for you to enter your
-database connection details. You can also supply these directly:
+### Running the Report
+
+The report can be run by running the following code.
+
+- The date range that you need to generate the report should be give
+  with *start_date* and *end_date*.
+- The acceptable tolerance for measurement checks can be given using the
+  *measurement_tolerance*
 
 ``` r
-library(d.inspectEHR)
+library(ccaa.inspectEHR)
 
-render_report(
-  output = "DECOVID_report.html",
-  prams = list(
+render_summary_report(
+  output = "Summary_report.html",
```

Comment:
Doesn't this argument need to go as one of the params?

Reply:
This is done! README file modified